src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -171,6 +171,7 @@ public void keyboardShortcutsTest() {
         assertEquals(repoSelectorComboBox.getValue(), "dummy/dummy");
         // test shortcut when focus is on panel
         PlatformEx.waitOnFxThread();
+        sleep(1000);
```

Comment:
This one is just to make sure it is reliable on local. Seems to be very unreliable without sleep.